### ISSUE TITLE
Update/Paid version

### ISSUE THREAD

**Issue by vavavr00m** [Vianvolaeus/blender-matcap-generator#9]
Did you publish the paid version somewhere? If yes, can I have the link, please?

**Comment by Vianvolaeus** [comment]
Hi vavavro00m, 

At this time, no. If there's something in particular you were looking for or hoping for from it, let me know and I'll help out if I can. 

I'll update you when it does release. 

V

**Comment by vavavr00m** [comment]
I'm more curious to know what your plans are and what you meant by "more feature complete version." I've just discovered matcaps and still researching stuff.

**Comment by Vianvolaeus** [comment]
Most planned features for the update are in the repo's Github issues. 

As it stands, the matcap generator is pretty usable, but I had a handful of people say it was too difficult to use, so I intended on making it a little easier to work with, and I've noticed that a lot of people output a lot of 'default' matcaps, not changing materials, lights, light position, etc. 

If you've got a specific use-case for matcaps or any further questions / research, feel free to reach out.
